test/bases/renku_data_services/data_api/test_storage_v2.py

```diff
@@ -2,6 +2,32 @@
 
 import pytest
 
+from renku_data_services.users.models import UserInfo
+
+
+@pytest.fixture
+def project_members(member_1_user: UserInfo, member_2_user: UserInfo) -> list[dict[str, str]]:
+    """List of a project's members."""
+    return [{"id": member_1_user.id, "role": "viewer"}, {"id": member_2_user.id, "role": "owner"}]
+
+
+@pytest.fixture
+def project_owner_member_headers(member_2_headers) -> dict[str, str]:
+    """Authentication headers for a normal project owner user."""
+    return member_2_headers
+
+
+@pytest.fixture
+def project_non_member_headers(unauthorized_headers) -> dict[str, str]:
+    """Authentication headers for a user that isn't a member of a project."""
+    return unauthorized_headers
+
+
+@pytest.fixture
+def project_normal_member_headers(member_1_headers) -> dict[str, str]:
+    """Authentication headers for a user that isn't a member of a project."""
+    return member_1_headers
+
```

Comment:
Because I was not looking forward to renaming all the parameters in all tests. And there were tests that were merged from another PR and I was resolving conflicts. But I will do this now.

Reply:
Ralf I would like to leave this as is. These fixtures came from the PR about storage that we just merged recently. And they were adding fully separate random users and not the ones we already have properly plugged in everywhere from conftest.py. So I just quickly changed this when I was fixing the tests after updating my PR against main.

I would like to leave these because they succinctly summarize which users have which roles in the projects. Otherwise it is a bit strange when you see an assertion that `member_1_headers` should make you a project member. This makes it clearer. And it is easier to change things here all in one place.